stdlib/src/utils/span.mojo

```diff
@@ -224,3 +224,20 @@ struct Span[
         """
 
         return self._data
+
+    @always_inline
+    fn copy_from[
+        lifetime: MutableLifetime, //
+    ](ref [_]self: Span[T, lifetime], ref [_]other: Span[T, _]):
```

Comment:
```suggestion
    ](inout self: Span[T, lifetime], other: Span[T, _]):
```

**Question** Does this work if we leave out these `ref` specifications?

`inout self` is more accurate than `ref [_] self` since `self` _must_ be mutable for this method to work.

And likewise `borrowed other` (leaving the `borrowed` implicit) is valid too because this method never mutates `other`.